### ISSUE TITLE
VSCE: Unable to list `SIT-parms` for a region with self-signed certs

### ISSUE THREAD

**Issue by zFernand0** [zowe/cics-for-zowe-client#124]
**Describe the bug**

Having rejectUnauthorized in the CICS profile (or base profile) does not matter when you try to show the SIT Parameters for the region

**Expected and actual results**

Expected to get the results of the following CICS reosurce
- Endpoint: `CICSSystemParameter`
- Parmameters: `PARMSRCE(COMBINED) PARMTYPE(SIT)`

Resulted in the following error:
![image](https://github.com/zowe/cics-for-zowe-client/assets/37381190/3a973eef-5c35-41e6-9505-688bbdefe331)

**Describe your environment**

ZE VSCE: v2.15.2
CICS VSCE: v2.3.5
SCS: Yes

**Additional context**

N/A

**Comment by davenice** [comment]
@AndrewTwydell - I think you've had a poke at debugging this?

**Comment by davenice** [comment]
Wondering if this is related to #159 - worth checking when you have a prototype fix for that, @AndrewTwydell.

**Comment by AndrewTwydell** [comment]
Yep, seems my dev branch to fix the rejectUnauthorized bugs also fix this. Will reference this issue when I open a PR

<img width="550" alt="Screenshot 2024-12-13 at 4 45 37 PM" src="https://github.com/user-attachments/assets/f39fe0c3-7753-418a-aaf1-8c78746ae186" />
